### PR TITLE
Volto 13

### PR DESCRIPTION
**Pull request by giuliaghisini**
Updated to Volto13.

Important!

- listingOptions conf and component no longer exists in Volto. They have been replaced by **listing variations** and **schemaEnhancer** and so src/components/ItaliaTheme/Blocks/Listing/Options was removed
- italiaConfig has been splitted into several parts because it was starting to get too complex
- removed some customizations because some configurable variables have been introduced in the Volt config (for example 'showTags')
- removed customization for LoginForm because it was the same of Volto.

### CODE REVIEW

File: src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx [hunk 1]
```diff
@@ -137,15 +138,31 @@ const ListingBody = React.memo(
...
+    let templateConfig;
 
-    let templateName =
-      data.template && !!templateConfig[data.template]
-        ? data.template
-        : 'default';
+    // Legacy support if template is present
+    if (data.template && !data.variation) {
+      const variations =
+        config.blocks?.blocksConfig['listing']?.variations || [];
```
Comment (nzambello) [inline src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx:147]: Non dovrebbe essere `templates`?
```suggestion
      const variations =
        config.blocks?.blocksConfig['listing']?.templates || [];
```

Comment (giuliaghisini) [reply src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx:147]: no no è variations.. lo hanno chiamato cosi...

Comment (nzambello) [reply src/customizations/volto/components/manage/Blocks/Listing/ListingBody.jsx:147]: okok capito

**Review by nzambello — APPROVED** [review]

**Review by pnicolli — APPROVED** [review]

File: src/config/Blocks/listingVariations.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+import SliderTemplateSkeleton from '@italia/components/ItaliaTheme/Blocks/Listing/TemplatesSkeletons/SliderTemplateSkeleton';
+
+import GridGalleryTemplate from '@italia/components/ItaliaTheme/Blocks/Listing/GridGalleryTemplate';
+import GridGalleryTemplateSkeleton from '@italia/components/ItaliaTheme/Blocks/Listing/TemplatesSkeletons/GridGalleryTemplateSkeleton';
+
+import BandiInEvidenceTemplate from '@italia/components/ItaliaTheme/Blocks/Listing/BandiInEvidenceTemplate';
+import BandiInEvidenceTemplateSkeleton from '@italia/components/ItaliaTheme/Blocks/Listing/TemplatesSkeletons/BandiInEvidenceTemplateSkeleton';
+
+import AmministrazioneTrasparenteTablesTemplate from '@italia/components/ItaliaTheme/Blocks/Listing/AmministrazioneTrasparenteTablesTemplate';
+import AmministrazioneTrasparenteTablesTemplateSkeleton from '@italia/components/ItaliaTheme/Blocks/Listing/TemplatesSkeletons/AmministrazioneTrasparenteTablesTemplateSkeleton';
```
Comment (nzambello) [inline src/config/Blocks/listingVariations.js:37]:
```suggestion
// import AmministrazioneTrasparenteTablesTemplate from '@italia/components/ItaliaTheme/Blocks/Listing/AmministrazioneTrasparenteTablesTemplate';
// import AmministrazioneTrasparenteTablesTemplateSkeleton from '@italia/components/ItaliaTheme/Blocks/Listing/TemplatesSkeletons/AmministrazioneTrasparenteTablesTemplateSkeleton';
```

File: src/config/Blocks/listingVariations.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  {
+    id: 'amministrazioneTrasparenteTablesTemplate',
+    isDefault: false,
+    title: 'Tabelle Amministrazione Trasparente',
+    template: AmministrazioneTrasparenteTablesTemplate,
+    skeleton: AmministrazioneTrasparenteTablesTemplateSkeleton,
+    schemaEnhancer: ({ schema, formData, intl }) => {
+      /*let pos = */ addDefaultOptions(schema, formData, intl);
+      return schema;
+    },
```
Comment (nzambello) [inline src/config/Blocks/listingVariations.js:197]:
```suggestion
//  {
//    id: 'amministrazioneTrasparenteTablesTemplate',
//    isDefault: false,
//    title: 'Tabelle Amministrazione Trasparente',
//    template: AmministrazioneTrasparenteTablesTemplate,
//    skeleton: AmministrazioneTrasparenteTablesTemplateSkeleton,
//    schemaEnhancer: ({ schema, formData, intl }) => {
//      /*let pos = */ addDefaultOptions(schema, formData, intl);
//      return schema;
//    },
```

**Review by nzambello — APPROVED** [review]

**Pull request merged**